packages/backend/package.json

```diff
@@ -47,6 +47,7 @@
     "@backstage/plugin-search-backend-module-techdocs": "^0.1.23",
     "@backstage/plugin-search-backend-node": "^1.2.23",
     "@backstage/plugin-techdocs-backend": "^1.10.5",
+    "@qshift/plugin-api-backend": "^0.1.0",
```

Comment:
"@qshift/plugin-api-backend": "^0.1.0", should be "@qshift/plugin-api-backend": "workspace:^",